src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin Page</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+	<nav>
```

Comment:
Though there is no style guide for HTML, it's good to have the indentation match for opening and closing brackets and be consistent. Something like 2 spaces for indentation.